source/macos/AdaptiveCards/AdaptiveCards/Utils/BundleUtils.swift

```diff
@@ -0,0 +1,25 @@
+import AppKit
+
+let bundleIdentifier = "com.test.test.AdaptiveCards"
+
+class BundleUtils {
+    static func getImage(_ imageName: String) -> NSImage? {
+        let index = imageName.lastIndex(of: ".") ?? imageName.endIndex
+        let fileType = String(imageName[imageName.index(after: index)...])
+        let name = String(imageName[...imageName.index(before: index)])
+        guard let bundle = Bundle(identifier: bundleIdentifier),
```

Comment:
Not really comfortable of doing these String operations here as they're fragile for edges cases. :/
May be you can simply have 2 params like the native definition.
```suggestion
    static func getImage(_ imageName: String, ofType type: String) -> NSImage? {
        guard let bundle = Bundle(identifier: bundleIdentifier),
```